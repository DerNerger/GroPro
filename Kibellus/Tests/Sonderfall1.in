;**************************
;     Sonderfall 1          
;**************************
Ei
Eisen
Eisenerz