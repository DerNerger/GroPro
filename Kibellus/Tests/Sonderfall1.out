;**************************
;     Sonderfall 1          
;**************************
Eingelesene Woerter:
EI, EISEN, EISENERZ

** Raetsel nicht versteckt

EISENERZ

** Raetsel versteckt

EISENERZ

Kompaktheitsmass: 8